### PR TITLE
adding cron_task for repl_monitor

### PR DESCRIPTION
**Pull request by kkonnov**

### CODE REVIEW

File: tasks/cron_jobs.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+- name: Removing old records in the repl_monitor table
+  cron:
+     name: "Removing repl_monitor records older than 4 days"
```
Comment (UnderGreen) [inline tasks/cron_jobs.yml]: This message will be for all users? Maybe need a simple message like 'Removing repl_monitor old records'

File: defaults/main.yml [hunk 2]
```diff
@@ -70,3 +70,22 @@ repmgr_promote_command: repmgr standby promote -f {{ repmgr_conf }}
...
+# Cron variables:
+
+repmgr_cron: []
+
+# repmgr_cron:
+#   hour: 4
+#   minute: 0
+#   day: *
+#   job: "psql -U postgres repmgr -c \"DELETE * FROM repmgr_test.repl_monitor WHERE last_monitor_time < current_timestamp - interval '4d'\""
+#   name: ""
```
Comment (UnderGreen) [inline defaults/main.yml]: You don't have variable name in task.

**Comment by kkonnov** [comment]
Ok. But why? This description explains what exactly task do. I suppose if someone need to change task parameters, he could change the description easily as well.

**Comment by UnderGreen** [comment]
Yes, sure. But again.. You don't have variable name inside cron task.

**Comment by kkonnov** [comment]
renewed. Should I create new branch once again? :)

**Pull request merged**